src/main/java/org/wildfly/naming/client/WildFlyRootContext.java

```diff
@@ -98,9 +98,6 @@ public NameParser getNativeNameParser() {
     }
 
     protected Object lookupNative(final Name name) throws NamingException {
-        if (name.isEmpty()) {
```

Comment:
By specification, if the name is empty, a new copy of the current context must be returned.  It is the intention that the root context be returned.